src/analysis/convex/body.lean

```diff
@@ -31,7 +30,8 @@ convex, convex body
 open_locale pointwise
 open_locale nnreal
 
-variables (V : Type*) [seminormed_add_comm_group V] [normed_space ℝ V]
+variables (V : Type*) [topological_space V] [add_comm_group V] [has_continuous_add V]
+  [module ℝ V] [has_continuous_smul ℝ V]
 
 /--
 Let `V` be a normed space. A subset of `V` is a convex body if and only if
```

Comment:
Can you also fix the docstring (and the other place in the file where a norm is mentioned)?